src/modules/map/components/cesium/CesiumInternalLayer.vue

```diff
@@ -65,12 +68,14 @@ export default {
             type: Number,
             default: null,
         },
+        projection: {
+            type: CoordinateSystem,
+            default: DEFAULT_PROJECTION,
```

Comment:
I would try to avoid using the default projection whenever possible and make it required to avoid any confusion